sensing/autoware_pointcloud_preprocessor/config/passthrough_filter_uint16_node.param.yaml

```diff
@@ -0,0 +1,7 @@
+/**:
+  ros__parameters:
+    filter_limit_min: 0
+    filter_limit_max: 127
+    filter_field_name: "ring"
```

Comment:
I think this would be channel now (applies to the schema as well)

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/8472/commits/ee7671cd00be67775148ca1782cb1b66b2dba250 and https://github.com/autowarefoundation/autoware.universe/pull/8472/commits/3efc46d82f0efed05259412f18660421257c84cf